targets/HUDL/write_to_display.cpp

```diff
@@ -12,11 +12,11 @@
 namespace IO = EVT::core::IO;
 namespace time = EVT::core::time;
 
-constexpr uint32_t SPI_SPEED = SPI_SPEED_4MHZ;// 4MHz
-constexpr uint8_t deviceCount = 1;
+const uint32_t SPI_SPEED = SPI_SPEED_4MHZ;// 4MHz
+const uint8_t deviceCount = 1;
```

Comment:
change const unit8_t deviceCount =1 to captitals since it is a constant. ie. const unit8_t DEVICECOUNT = 1 
Need to then replace name in rest of main()
(๑ↀᆺↀ๑)

Reply:
![image](https://user-images.githubusercontent.com/30483634/194441656-e2f1789d-ad3d-47d0-976d-556a009e0436.png)

Reply:
DEVICE_COUNT
ლ(●ↀωↀ●)ლ